caf-audit/src/main/java/com/hpe/caf/auditing/elastic/ElasticAuditEventBuilder.java

```diff
@@ -97,7 +99,8 @@ public void setTenant(String tenantId){
         }
         this.tenantId = tenantId.toLowerCase();
 
-        // TODO CAF-2610 - Create Elasticsearch index for the specified tenant if it does not already exist.
+        // Create Elasticsearch index for the specified tenant.
+        indexManager.getIndex(ES_INDEX_PREFIX.concat(this.tenantId));
```

Comment:
@glucy I revisited your original comment on index names. I now see your use of \<tenantid\>_audit in the index name.